unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackPreviewPanel.cs

```diff
@@ -12,9 +12,20 @@ namespace DCL.Backpack
     public class BackpackPreviewPanel : BaseComponentView
     {
         private const string RESET_PREVIEW_ANIMATION = "Idle";
+        private const float CAMERA_MIN_X_LIMIT = 0f;
+        private const float CAMERA_MAX_X_LIMIT = 0f;
+        private const float CAMERA_MIN_Y_LIMIT = 0f;
+        private const float CAMERA_MAX_Y_LIMIT = 1.7f;
+        private const float CAMERA_MIN_Z_LIMIT = 1.3f;
+        private const float CAMERA_MAX_Z_LIMIT = 3.5f;
+        private const float CAMERA_ZOOM_CENTER = 1.4f;
+        private const float CAMERA_ZOOM_BOTTOM_MAX_OFFSET = 1.1f;
+        private const float CAMERA_ZOOM_TOP_MAX_OFFSET = 0.3f;
```

Comment:
You can use a static readonly Bounds here, it will be way cleaner.

Reply:
It was already done here: https://github.com/decentraland/unity-renderer/pull/5162/commits/aedad45cd33b4edb672c3c41ac6d555a365eed5b